docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -170,33 +115,33 @@ gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5050 ! "application/x-rtp-s
 gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5080 ! audio/x-raw,format=S16LE,rate=16000,channels=1 ! pulsesink
 ```
 
-# Start SDL Core
-In the build folder
+## Video Streaming States
 
-```
-cd bin
-./start.sh
-```
-!!! NOTE
-If you want to use a USB connection, you must run
-```
-sudo ./start.sh
-```
-!!!
+This section describes how Core manages the streaming states of mobile applications. Only one application may stream video at a time, but audio applications may stream while in the LIMITED state with other applications.
+
+When an app is moved to HMI level `FULL`:
+- All non-streaming applications go to HMI level `BACKGROUND`
```

Comment:
The bullet points in https://staging.smartdevicelink.com/en/guides/pull_request/core_6_1/0be31839052bea6e9c529af051d82e2d35201ebc/integrating-your-hmi/sdl-core-and-hmi-communication/#request look somewhat better. I'd suggest trying that format for the bullet points.

```suggestion
  * All non-streaming applications go to HMI level `BACKGROUND`
```

Reply:
definitely. changed in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/b07a1990e1799725a09996753b1db4bba73b664b

Reply:
Thanks for making the change. Unfortunately the result isn't quite what I was expecting.
![Screen Shot 2020-03-04 at 3 00 54 PM](https://user-images.githubusercontent.com/10703941/75918141-2a9fb080-5e29-11ea-8f6e-939b4fc08cd4.png)

Not sure why the doc server parses it like that. I did notice that there is a new line before the first bullet in https://smartdevicelink.com/en/guides/pull_request/core/c3c6b5e47bc41ba52003b393711775eae3a40ab0/integrating-your-hmi/sdl-core-and-hmi-communication/#request.  If you want to experiment with it, you could try making that change. Otherwise I think it's okay to change it back to `-`